tests/ui/lint/fn_null_check.stderr

```diff
@@ -0,0 +1,67 @@
+warning: checking for function pointer nullability does nothing as they are assumed to be non-null
+  --> $DIR/fn_null_check.rs:6:8
+   |
+LL |     if (fn_ptr as *mut ()).is_null() {}
+   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = help: wrap the function pointer inside an `Option` and use `Option::is_none` to check for null pointer value
+   = note: `#[warn(incorrect_fn_null_checks)]` on by default
+
+warning: checking for function pointer nullability does nothing as they are assumed to be non-null
```

Comment:
"does nothing" sounds a bit odd here... what about something like "function pointers cannot be null, so this check will always come out as non-null`?

Reply:
Sure, sound good but instead of "so this check" what about "so checking for their nullability" ?

<details>

> warning: function pointers cannot be null, so **checking for their nullability** will always come out as non-null

</details>